templates/terraform/resource.html.markdown.erb

```diff
@@ -82,6 +83,23 @@ To get more information about <%= object.name -%>, see:
 
 ~> **Warning:** <%= object.docs.warning -%>
 <%- end -%>
+<%- if !sensitive_props.empty? -%>
+<%-
+  sense_props = sensitive_props.map! {|prop| "`"+prop.name.underscore+"`"}.uniq
+  if sense_props.length == 1
+    sense_props = sense_props[0]
+  elsif sense_props.length == 2
+    sense_props = sense_props[0]+" and "+sense_props[1]
+  else
+    sense_props = sense_props.slice(0,sense_props.length-1).join(", ") + " and " + sense_props[sense_props.length-1]
+  end
+-%>
+
+~> **Warning:** All arguments including <%= sense_props -%> will be stored in the raw
+state as plain-text. [Read more about sensitive data in state](/docs/state/sensitive-data.html).
+Because the API does not return the sensitive key value,
```

Comment:
This bit isn't _always_ true, it sometimes is returned. I'd lean more towards omitting it than finding out where it is or isn't.